app/models/post.rb

```diff
@@ -22,6 +22,14 @@ def summary
   end
 
   def user_name
-    "#{user.first_name} #{user.last_name}"
+    "#{user.user_name}"
+  end
+
+  def like
+    if Like.where(post_id: self.id).count == 1
```

Comment:
Can you use rails #pluralize method https://apidock.com/rails/String/pluralize instead

I'm sure you have access to the association method `post#likes` Can you not use this instead?

```ruby
"Like".pluralize(likes.count)
```

Reply:
something like this anyway

Reply:
And make sure both outcomes are tested

Reply:
pluralize method is nifty.